pkg/operator/config/fs_importer.go

```diff
@@ -15,30 +16,45 @@ type FSImporter struct {
 	fs fs.FS
 
 	cache map[string]jsonnet.Contents
+	paths []string
 }
 
 // NewFSImporter creates a new jsonnet VM Importer that uses the given fs.
-func NewFSImporter(f fs.FS) *FSImporter {
+func NewFSImporter(f fs.FS, paths []string) *FSImporter {
 	return &FSImporter{
 		fs:    f,
 		cache: make(map[string]jsonnet.Contents),
+		paths: paths,
 	}
 }
 
 // Import implements jsonnet.Importer.
 func (i *FSImporter) Import(importedFrom, importedPath string) (contents jsonnet.Contents, foundAt string, err error) {
-	cleanedPath := path.Clean(
-		path.Join(path.Dir(importedFrom), importedPath),
-	)
-	cleanedPath = strings.TrimPrefix(cleanedPath, "./")
 
+	tryPaths := append([]string{importedFrom}, i.paths...)
```

Comment:
This will default to checking files relative to the current directory and then relative to each path in `i.paths`.